docs/website/docs/dlt-ecosystem/file-formats/_set_the_format.mdx

```diff
@@ -0,0 +1,23 @@
+import CodeBlock from '@theme/CodeBlock';
+
+<span>There are several ways of configuring dlt to use <code>{props.file_type}</code> file format for normalization step and to store your data at the destination:</span>
+
+1. <span>You can set the <code>loader_file_format</code> argument to <code>{props.file_type}</code> in the run command:</span>
+
+<pre>
```

Comment:
no way to use language tags here?
```py
```
not rendered correctly? the problem is that this won't be linted by our tooling

Reply:
Added language tags. 

Sorry, I had to play around with tags to be able to use props, so I just forgot about this.